app/users-page/components/users-grid.tsx

```diff
@@ -0,0 +1,66 @@
+import { GridColumn } from '@progress/kendo-react-grid';
+
+import { observer } from 'mobx-react-lite';
+import * as React from 'react';
+import { useEffect, useMemo } from 'react';
+
+import { getEditAndDeleteActionCell } from '../../common/components/kendo-grid/editable-cell/get-edit-and-delete-action-cell';
+import { getSelectCell } from '../../common/components/kendo-grid/editable-cell/get-select-cell';
+import { PasswordCell } from '../../common/components/kendo-grid/editable-cell/password-cell';
+import { KendoGrid } from '../../common/components/kendo-grid/kendo-grid';
+import { InMemoryDataSource, KendoGridState } from '../../common/components/kendo-grid/kendo-grid-state';
+import { getEnumValues } from '../../common/utils/form-helpers';
+import { Users } from '../../mocks/user';
+import { Role, User } from '../../user/models';
+
+type UsersGridState = KendoGridState<User, User['id']>;
+
+export const UsersGrid = observer(() => {
+    const gridState = useUsersGridState();
+
+    const ActionCell = useActionCellMemo(gridState);
+
+    return (
+        <KendoGrid
+            gridState={gridState}
+            selectable
+            sortable
+            groupable={{ footer: 'visible' }}
```

Comment:
It isn't required in this case.
```suggestion
            groupable
```